Description for 'Consumption Typ' specified (#42793)
home-assistant/home-assistant.github.io: source/_integrations/shelly.markdown

## Consumption type (generation 2+)

> Shelly device relays are added to Home Assistant by default as `switch` entities. A relay can be added as a `light` entity if **Consumption Type** value in the Shelly WebUI of the device is set to `light`.  

Reviewer: ```suggestion
Shelly device relays are added to Home Assistant by default as `switch` entities. A relay can be added as a `light` entity if **Consumption type** value in the Shelly WebUI of the device is set to `light`.  
```

> **Home** >> **Switches and attached inputs** >> **Consumption type** (Output settings)

Reviewer: `Consumption type` and `Output settings` are two different sections in the UI

<img width="1247" height="633" alt="Image" src="https://github.com/user-attachments/assets/f0118893-2f5b-41cb-b118-5a9f955dbe24" />

```suggestion
**Home** >> **Switches and attached inputs** >> **Consumption type**
```

Reviewer 2: I think it is to show that it is under the output settings section (what I marked in yellow) but maybe it is confusing and we should remove it.
<img width="1241" height="613" alt="image" src="https://github.com/user-attachments/assets/93bee2de-e829-4d07-9017-7d96790d0d9b" />

Reviewer: TBH I didn't notice this first (yellow) `output settings` 😅

But, yes I think we should remove this.

Author: Which device and firmware did you use to check?
Perhaps the gui on the different devices is not the same. In the case my additional note is perhaps missleading.
I have Shelly Plus1/Shelly Plus1PM with firmware 1.7.1 and here it looks different:
<img width="670" height="413" alt="Shelly1PM_Consumption_Settings" src="https://github.com/user-attachments/assets/5d63dd78-3dbd-42f5-a6c4-b2851adcac7d" />

Reviewer 2: I guess it is related to the specific device (Plus Plug S), I checked few devices with different firmware and they look like the picture you poste from 1.7.1.
I think we can still remove the "(Output settings)" so that we don't confuse users with the sub-menu of input/output settings.

> Shelly device relays are added to Home Assistant by default as `switch` entities. A relay can be added as a `light` entity if **Consumption type** value in the Shelly WebUI of the device is set to `light`.  
> **Home** >> **Switches and attached inputs** >> **Consumption type** (Output settings)

Reviewer 3: ```suggestion
Shelly device relays are added to Home Assistant by default as `switch` entities. A relay can be added as a `light` entity if **Consumption type** value in the Shelly WebUI of the device is set to `light`, under 
**Home** > **Switches and attached inputs** > **Consumption type**.
```
If I understand correctly, (Output settings) needs to be removed, and the PR could be merged?

Reviewer 2: I am not sure if this PR makes it more clear or just goes backward (we just added this text in recent PR).
Picture from 1.7.1:
<img width="305" height="389" alt="image" src="https://github.com/user-attachments/assets/71bd725a-174b-4184-9685-e5e10badd59b" />
And 1.3.0 looks the same:
<img width="261" height="351" alt="image" src="https://github.com/user-attachments/assets/c6364d04-35b6-4664-b3d4-e1575b489ab0" />

The `Output settings` refers to the relevant section in the page.
@bieniu what do you think?

Reviewer: In the plugs UI there is an `output settings` section and option.

![image](https://github.com/user-attachments/assets/aa446973-7dcf-423f-8bda-8914f6dc2942)

In my opinion we should remove `(Output settings)` because it can be confusing.

> **Home** >>> **Switches and attached inputs** >>> **Consumption type** (Output settings)

Reviewer: In other places in the documentation we use `>>`, not `>>>`
```suggestion
**Home** >> **Switches and attached inputs** >> **Consumption type** (Output settings)
```
